### PR TITLE
🚧Refactor unit test assertions

### PR DESCRIPTION
**Pull request by SyedMSawaid**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title matches the title of the associated issue.

- [x] A `preview` label has been added to the pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] Project assigned matches the project of the associated issue.

- [x] Milestone assigned matches the milestone of the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out, and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: Testing/VelaptorTests/OpenGL/WindowSizeEventArgsTests.cs [hunk 1]
```diff
@@ -17,15 +18,16 @@ public class WindowSizeEventArgsTests
...
 
             // Act
             var actualWidth = eventArgs.Width;
             var actualHeight = eventArgs.Height;
 
             // Assert
-            Assert.Equal(123, actualWidth);
-            Assert.Equal(456, actualHeight);
+            actualHeight.Should().Be(heightArg);
+            actualWidth.Should().Be(widthArg);
```
Comment (CalvinWilkinson) [inline Testing/VelaptorTests/OpenGL/WindowSizeEventArgsTests.cs:30]: This works just fine as it is right now. :)

Just a couple of things though.  For the purpose of keeping the code as simple and short as possible, I would just hard code the values into the `Be()` methods instead of creating a hard-coded tuple value that deconstructs into 2 separate variables.

**I suggest the example below:**
```cs
            // Arrange
            var eventArgs = new WindowSizeEventArgs(123, 456);

            // Act
            var actualWidth = eventArgs.Width;
            var actualHeight = eventArgs.Height;

            // Assert
            actualWidth.Should().Be(123);
            actualHeight.Should().Be(456);
```

Just a quick couple of tips.  Usually, when it comes to unit tests in general, it is better to be a little more verbose, have a little more layout such as the AAA setup like you see here, and have the code be as simple as possible.

The benefit of simple and easy-to-understand tests makes it a lot easier in the future to see why things are broken and what the expected behavior is.  Unit tests should act like documentation of your code in the sense of dependencies and behavior.

**Review by CalvinWilkinson — CHANGES_REQUESTED** [review]

**Comment by SyedMSawaid** [comment]
@CalvinWilkinson I updated the code. Kindly check if it satisfy your coding standards.

**Comment by CalvinWilkinson** [comment]
@SyedMSawaid Do you have the ability to change the title?  I want to say that you do.
The reason I ask is that the title for pull requests needs to match exactly the title of the associated issue.

No big deal.  I will update that right now but I would like to know if outside contributors have that ability.  
:)  
Cheers!!

**Review by CalvinWilkinson — APPROVED** [review]
